source/content/guides/secrets/05-local-development.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Local development using Pantheon Secrets
+description: Developing locally presents some unique challenges once Pantheon Secrets are built into your workflow. These are some tips to help you get past struggling with trying to reproduced secret behavior while developing locally.
+terminuspage: true
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/local-development
+reviewed: "2024-05-01"
+---
+## Local Environment Usage
+
+The SDK includes a `CustomerSecretsFakeClient` implementation that is used when the SDK runs outside of Pantheon infrastructure. This client uses a secrets json file to build the secrets information emulating what happens on the platform using the Secrets service.
```

Comment:
Be more specific about "The SDK". Note that this advice applies both to the Customer Secrets PHP SDK directly, and also to the Pantheon Key module plugin, which uses the same SDK.

Reply:
> The [Pantheon Secrets SDK](https://github.com/pantheon-systems/customer-secrets-php-sdk) includes a `CustomerSecretsFakeClient` implementation that is used when the SDK runs outside of Pantheon infrastructure.